Packs/Core/Integrations/CortexCoreIR/CortexCoreIR.yml

```diff
@@ -4115,6 +4115,156 @@ script:
     - contextPath: Core.ScriptResult.results.command
       description: The command that was executed by the script.
       type: String
+  - arguments:
+    - description: String that identifies the indicator you want to insert into Cortex. Required when ioc_object is not in use.
+      name: indicator
+      required: false
+      isArray: false
+      type: String
+    - auto: PREDEFINED
+      description: Keyword identifying the type of indicator. Required when ioc_object is not in use.
```

Comment:
The keyword identifying the type of indicator. Required when the ioc_object is not in use.